central/globaldb/v2backuprestore/manager/manager.go

```diff
@@ -121,15 +121,20 @@ func (m *manager) LaunchRestoreProcess(ctx context.Context, id string, requestHe
 		return nil, err
 	}
 
-	if err := m.checkDiskSpace(totalSizeUncompressed); err != nil {
-		return nil, err
-	}
-
 	process, err := newRestoreProcess(ctx, id, requestHeader, handlerFuncs, data)
 	if err != nil {
 		return nil, err
 	}
 
+	if !process.postgresBundle {
+		if _, err := os.Stat(m.outputRoot); os.IsNotExist(err) {
```

Comment:
Can you please check the outputRoot as mounting point? Because we still need /var/lib/stackrox as a mounting point for centrals upgraded.